frontend/components/viewer/js/viewer.service.ts

```diff
@@ -239,22 +239,23 @@ export class ViewerService {
 	}
 
 	public handleUnityError = (message: string, reload: boolean, isUnity: boolean) =>  {
-
 		let errorType = '3D Repo Error';
 
 		if (isUnity) {
 			errorType = 'Unity Error';
 		}
 
-		this.DialogService.html(errorType, message, true)
-			.then(() => {
+		dispatch(DialogActions.showDialog({
+			title: errorType,
+			template: SimpleErrorDialog,
```

Comment:
What is the difference between `SimpleErrorDialog` and error dialog that already exists?